internal/bminventory/inventory_v2_handlers.go

```diff
@@ -0,0 +1,17 @@
+package bminventory
```

Comment:
Should all V2 API move here?
Why not without the V2UpdateHostInternal ?

Reply:
this file will contain only V2 handlers. in MGMT-7506 we will move the V2 APIs that are already in the inventory.go.  The final goal is to split inventory.go to 2 files: pne for handlers ( Installer API), and one for Internal API